optuna/visualization/_param_importances.py

```diff
@@ -29,6 +29,8 @@ class _ImportancesInfo(NamedTuple):
     importance_values: List[float]
     param_names: List[str]
     importance_labels: List[str]
+    target_name: str
+    hover_template: List[str]
```

Comment:
Since this is for plotly, why not calculate it in _get_importances_plot of plotly?